src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.domain
+
+class LottoNumber private constructor(private val number: Int) {
+    fun getNumber() = number
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class LottoNumber private constructor(val number: Int) {
```